.github/workflows/ci.yml

```diff
@@ -254,12 +251,12 @@ jobs:
         run: |
           if [ "${ADL_NAME}" != "" ]; then
             source plugin/trino-hive-hadoop2/conf/hive-tests-${{ matrix.config }}.sh &&
-              plugin/trino-hive-hadoop2/bin/run_hive_adl_tests.sh
+              MAVEN_TEST="${MAVEN_TEST} ${GIB_OPTS}" plugin/trino-hive-hadoop2/bin/run_hive_adl_tests.sh
```

Comment:
Would it work if MAVEN_TEST was redefined in the whole job?

Reply:
We can add job specific env vars via `env:` - I'll need to see if it allows `concat`.

Reply:
See https://github.com/trinodb/trino/pull/11569/commits/994f912e1f5df179fefbf03c3468eb9affb75b00 for an attempt, not sure if it works as expected, we'll know.

Reply:
Nope, doesn't work. `env` cannot refer to other env values.